rpa-recorder/6.2/modules/ROOT/pages/user-interface.adoc

```diff
@@ -149,7 +149,7 @@ Use the following settings to configure RPA Recorder:
 * *Directory*: Specifies the local folder for storing documents (BPMN, PDF and XES) and application logs.
 * *Create Process Documentation (PDF)*: If checked, a process documentation file is saved as a PDF after the recording session stops.
 * *Create Process Mining files (XES)*: If checked, a XES process file is generated after the recording session stops.
-* *Generate debug file with a list of raw recorded events*: If checked, a debug file is generated after the recording session stops.
+* *Generate debug file (contains recorded events, window handles and system data)*: If checked, a debug file is generated after the recording session stops.
```

Comment:
The Chicago comma seems to be missing. This should read:
"...contains recorded events, window handles**,** and system data..."